src/azul/plugins/metadata/hca/transform.py

```diff
@@ -350,17 +354,22 @@ def for_id(cls, submitter_id: str) -> Optional['Submitter']:
         except KeyError:
             return None
 
+    @classmethod
+    def for_title(cls, title: str) -> Optional['Submitter']:
+        try:
+            submitter_id = cls.id_by_title[title]
+        except KeyError:
+            return None
```

Comment:
As I said earlier today, this need to fail, so that we learn immediately that our list of submitters is out of sync.